### ISSUE TITLE
DatePicker: Time inputs can not be focussed by keyboard when inline

### ISSUE THREAD

**Issue by jepsar** [primefaces/primefaces#8430]
```xhtml
<p:datePicker id="inline" value="#{calendarJava8View.date1}" inline="true"
              showTime="true" timeInput="true"/>
```

**Comment by melloware** [comment]
You might want to link this ticket with there is another one about Datepicker 509 compliance which is basically the whole component is not keyboard accessible

**Comment by jepsar** [comment]
#4685 you mean.

**Comment by melloware** [comment]
Yep!

**Comment by melloware** [comment]
@jepsar I can focus on the inputs in inline mode fine?   I just can't select them with the keyboard. Which is basically #4685  

Are you saying you can't even TAB to them?  Because I can TAB to them fine?

**Comment by jepsar** [comment]
@melloware I've updated the description of the issue with a reproducer. Just tested it with the latest version and it is not working for me. I can focus the arrows, but I cannot focus the inputs.

The other issue is broad. I guess it mainly is about popup, as inline is pretty okay already.

**Comment by melloware** [comment]
Ohhhh sorry I thought you meant the arrows now the inputs.  Thanks for the update!

**Comment by melloware** [comment]
to make this consistent with all the other tabindexes in DatePicker they all should be 0 so you tab in order.  That was the cause of this issue

**Comment by jepsar** [comment]
I think that it was done intentionally to fix a Firefox issue. I'll need to dive in the history to check, but I think it was added by @christophs78

**Comment by melloware** [comment]
yeah the whole component needs 508 review in https://github.com/primefaces/primefaces/issues/4685

**Comment by melloware** [comment]
Tested in Firefox and it seems OK so let me know what you find out.  PrimeReact does not have this feature so I have nothing to compare against.
